trin-core/src/portalnet/types/messages.rs

```diff
@@ -417,6 +417,22 @@ impl Content {
     }
 }
 
+impl TryInto<Value> for Content {
+    type Error = MessageDecodeError;
+
+    fn try_into(self) -> Result<Value, Self::Error> {
+        if let Content::ConnectionId(val) = self {
+            Ok(Value::String(val.to_string()))
+        } else if let Content::Content(val) = self {
+            Ok(Value::String(encode(val.to_vec())))
+        } else if let Content::Enrs(val) = self {
+            Ok(Value::String(val[0].to_string()))
```

Comment:
Is there any specific reason we are returning only the first value of the `Vec<SzzEnr>`?

When the response is empty `Content::Enrs`, this is failing with:

```bash
thread 'tokio-runtime-worker' panicked at 'index out of bounds: the len is 0 but the index is 0', trin-core/src/portalnet/types/messages.rs:429:30
```
We need to handle this in a way to allow empty `Content::Enrs` value.

Reply:
Ahh good catch! This is just leftover sloppy code from the initial implementation phase. It's no longer failing on empty content. Also updated the jsonrpc result to show what kind of data was being returned 
```json
{
  ...
  "result": {
    "enrs": "[]"
  }
}```
instead of 
```json
{
  ...
  "result": "[]"
}```